src/Client.php

```diff
@@ -25,6 +25,14 @@
  */
 class Client
 {
+    public const LOG_DISABLED = 0b0000;
```

Comment:
These are some special params of the ES?

Reply:
No, this is just a way to write readable flags. `0b0000` is unsigned int 0, `0b0001` is unsigned int 1, `0b0010` is unsigned int 2, `0b0011` is unsingned int 3, etc.